### PR TITLE
correcting syntax errors in info blocks

### PR DESCRIPTION
**Pull request by somindatommy**
Correcting syntax errors in info blocks

### CODE REVIEW

File: en/docs/setup/changing-to-ibm-db2.md [hunk 1]
```diff
@@ -182,11 +182,11 @@ Apart from the basic configurations specified above, WSO2 Identity Server suppor
...
-    !!! info 
-        For more information on other parameters that can be defined in
-        the `<IS_HOME>/repository/conf/deployment.toml` file, see [Tomcat
-        JDBC Connection
-        Pool](http://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
+!!! info 
+    For more information on other parameters that can be defined in
+    the `<IS_HOME>/repository/conf/deployment.toml` file, 
+    see [Tomcat JDBC Connection
+    Pool](http://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
```
Comment (ayshsandu) [inline en/docs/setup/changing-to-ibm-db2.md]: Shall we point to Tomcat 9 document

File: en/docs/setup/changing-to-mssql.md [hunk 2]
```diff
@@ -181,11 +181,11 @@ Apart from above basic configurations WSO2 Identity Server supports advanced dat
...
-    !!! info 
-        For more information on other parameters that can be defined in
-        the `<IS_HOME>/repository/conf/deployment.toml` file, see [Tomcat
-        JDBC Connection
-        Pool](http://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
+!!! info 
+    For more information on other parameters that can be defined in
+    the `<IS_HOME>/repository/conf/deployment.toml` file, see [Tomcat
+    JDBC Connection
+    Pool](http://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
```
Comment (ayshsandu) [inline en/docs/setup/changing-to-mssql.md]: Shall we point to Tomcat 9 document? Do this in all applicable places

File: en/docs/setup/changing-to-ibm-db2.md (hunk 1 above)
Comment (somindatommy) [reply en/docs/setup/changing-to-ibm-db2.md]: Fixed : 3714c5859a2c616edf12d1dc94282501e2d97f71

File: en/docs/setup/changing-to-mssql.md (hunk 2 above)
Comment (somindatommy) [reply en/docs/setup/changing-to-mssql.md]: Fixed : 3714c5859a2c616edf12d1dc94282501e2d97f71

**Pull request merged**